Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM python:2.7-alpine
-RUN apk add --update openssl
+RUN apk add --update
```

Comment:
If we're not installing anything, then we can remove this line entirely.

Reply:
Fixed!